lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -206,6 +211,100 @@ def setUp(self):
             ]
         )
 
+    @mock.patch('xmodule.x_module.XModuleMixin.has_dynamic_children', mock.Mock(return_value='True'))
+    def test_view_redirect_if_entrance_exam_required(self):
```

Comment:
Yeah, I tried that as well -- ie, hitting the URL you provided directly --
after logging in I was redirected to the entrance exam page/content -- will
keep poking at it

On Fri, Mar 6, 2015 at 1:55 PM, Christina Roberts <notifications@github.com>
wrote:

> In lms/djangoapps/courseware/tests/test_entrance_exam.py
> <https://github.com/edx/edx-platform/pull/6997#discussion_r25967637>:
>
> > @@ -206,6 +211,100 @@ def setUp(self):
> >              ]
> >          )
> >
> > +    @mock.patch('xmodule.x_module.XModuleMixin.has_dynamic_children', mock.Mock(return_value='True'))
> > +    def test_view_redirect_if_entrance_exam_required(self):
>
> I directly hit the URL for a section outside of the entrance exam.
>
> As far as the score, if you go to
> http://studio.pr6997.m.sandbox.edx.org/settings/details/wew/wefe/wewe you
> will see that the exam score should be 80% at the moment, not 100%. I went
> back and forth a couple times before I wrote up the bug, so the current
> state does not match the repro steps.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/6997/files#r25967637>.
>



-- 
Matthew Drayer
Solutions Architect
edX | MIT
mattdrayer@edx.org